utils/utils.go

```diff
@@ -164,6 +164,22 @@ func DeleteLogs(logger *zap.Logger) {
 	}
 }
 
+func DeleteComposeFile(logger *zap.Logger) {
```

Comment:
This is a duplicate code, the same code is there for deleteLogs, please pass the fileName and change the method name. Throw an error in the function and handle it at the caller function.